### PR TITLE
SOC-1857 Clean up the module before starting work

### PR DESCRIPTION
**Pull request by garthwebb**
https://wikia-inc.atlassian.net/browse/SOC-1857

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php [hunk 1]
```diff
@@ -27,11 +27,13 @@ class FavoriteWikisModel extends WikiaModel {
 	 * @param User $user
 	 */
 	public function __construct( User $user ) {
+		parent::__construct();
```
Comment (jsutterfield) [inline extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php:36]: Minor: Is there a reason to call the parent constructor here? It doesn't look like we're using any of the helper values it initializes in this PR or in the other one.

Comment (garthwebb) [inline extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php:36]: I think its good form unless there is a specific reason not to.  Additionally, I often see code that does:

``` php
$wg = \F::app()->wg;
```

in classes that would give them that variable if they'd just inherited from parent.

Comment (jsutterfield) [inline extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php:36]: Sounds good :+1:

**Comment by jsutterfield** [comment]
LGTM

**Pull request merged**